src/octoprint/plugins/action_command_notification/__init__.py

```diff
@@ -120,6 +135,10 @@ def action_command_handler(self, comm, line, action, *args, **kwargs):
             return
 
         message = parameter.strip()
+
+        if self._filter.search(message):
+            return
```

Comment:
For debugging purposes, I think it would be a good idea to add a **debug** log output here, so that once enabled the log would clarify why certain notifications don't show up in the sidebar/the popups anymore.

```suggestion
            self._logger.debug(f"Notification matches filter regex: {message}")
            return
```